pkg/routeagent_driver/handlers/ovn/south_rules.go

```diff
@@ -31,21 +31,29 @@ func (ovn *Handler) handleSubnets(subnets []string, ruleFunc func(rule *netlink.
 	return nil
 }
 
-func (ovn *Handler) ruleForSouthTraffic(localSubnet, remoteSubnet string) (*netlink.Rule, error) {
-	_, dstCIDR, err := net.ParseCIDR(localSubnet)
-	if err != nil {
-		return nil, errors.Wrapf(err, "error trying to parse local subnet %q", localSubnet)
+func (ovn *Handler) programRule(dest, src string, tableId int) (*netlink.Rule, error) {
+	rule := netlink.NewRule()
+
+	if dest != "" {
+		_, dstCIDR, err := net.ParseCIDR(dest)
+		if err != nil {
+			return nil, errors.Wrapf(err, "error trying to parse toSubnet %q", dest)
+		}
+
+		rule.Dst = dstCIDR
 	}
 
-	_, srcCIDR, err := net.ParseCIDR(remoteSubnet)
-	if err != nil {
-		return nil, errors.Wrapf(err, "error trying to parse remote subnet %q", remoteSubnet)
+	if src != "" {
+		_, srcCIDR, err := net.ParseCIDR(src)
+		if err != nil {
+			return nil, errors.Wrapf(err, "error trying to parse fromSubnet %q", src)
+		}
+
+		rule.Src = srcCIDR
 	}
 
-	rule := netlink.NewRule()
-	rule.Dst = dstCIDR
-	rule.Src = srcCIDR
-	rule.Table = constants.RouteAgentHostNetworkTableID
+	rule.Table = tableId
+	rule.Priority = tableId
```

Comment:
```suggestion
	rule.Priority = tableId
```

May be we should just use a fixed priority for now, or accept a parameter for it?, setting the prioity as the table ID is confusing :)

Reply:
Ok, I understand now, but it's confusing. Can we document via comments at least?

I would be more explicit and separate the input of priority & table, even if they will be the same value
in the end.
```
 programRule(dest, src string, tableId, priority int) 
```

Can we do something like this later as a follow up?

Reply:
Sure, we can do that.